### ISSUE TITLE
Views content shows up twice when "Main Content" block is moved to another region

### ISSUE THREAD

**Issue by stpaultim** [backdrop/backdrop-issues#3397]
## Describe your issue or idea

I created a view of some test content and created a page display. 
When I navigated to the page, I could see my content showing up twice. 

### Steps to reproduce (if reporting a bug)

1) spin up fresh install of BackdropCMS 
2) create a couple of test posts
3) Create PAGE view of test posts, format = table, path sample.com/test-view
4) Go to sample.com/test-view (everything should be fine)
5) Go to Default Layout and move the Main Page Content to Bottom region. 
6) Go to sample.com/test-view and you should see the test view displayed twice. Once in Content region and once in Bottom region. 

### Actual behavior (if reporting a bug)

See above

### Expected behavior (if reporting a bug)

I expect to see the view displayed only once in whichever region where the Main Page Content block is.

**Comment by BWPanda** [comment]
Sounds like a duplicate of https://github.com/backdrop/backdrop-issues/issues/2691...?
